jax/_src/pallas/triton/pallas_call_registration.py

```diff
@@ -88,7 +88,7 @@ def pallas_call_lowering(
     print(grid_mapping)
 
   lowering_result = lowering.lower_jaxpr_to_triton_module(
-      jaxpr, (*in_shapes, *out_shapes), grid_mapping, name,
+      jaxpr, (*in_shapes, *out_shapes), grid_mapping, name, ctx.module_context.platforms[0]
```

Comment:
@gnecula is this the recommended way of getting the current platform?

Reply:
This code just changed, in #21799. Once you rebase there you will have access to a variable called `lowering_platform` which you should use.

Reply:
Rebase done